shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -32,6 +32,16 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
 }
 @end
 
+static bool ClipBoundsContainsPlatformViewBoundingRect(const SkRect& clipBounds,
```

Comment:
This needs a declaration comment.

It would be especially helpful to explain what the bounds and rect are supposed to be relative to.

Reply:
(And what the matrix is applied to.)